recipes/libinterpolate/all/conanfile.py

```diff
@@ -43,9 +43,6 @@ def requirements(self):
         self.requires("eigen/3.3.7", transitive_headers=True)
 
     def validate(self):
```

Comment:
```suggestion
    def validate(self):
        if Version(self.version) < "2.6.4" and self.settings.os != "Linux":
            raise ConanInvalidConfiguration(f"{self.ref} is not supported by {self.settings.os}; Try the version >= 2.6.4")
        if Version(self.version) >= "2.6.4" and self.settings.os not in ["Linux", "Windows"]:
            raise ConanInvalidConfiguration(f"{self.ref} is not supported by {self.settings.os}.")
```

If I understand correctly, Windows has been supported since 2.6.4, but not before. Plus, only Linux and Windows are supported now, other OS like FreeBSD and Mac are not supported yet.

Reply:
Yes, good catch. I wasn't thinking about the same recipe being used for the older versions. The library may build on the other systems but I can't test it, so they should be disabled until I am able to confirm.

Reply:
@CD3 if you can not confirm it, no problem, we can merge anyway, just add a TODO comment asking to check it in the future.

Reply:
No, as a comment in the code, like:

`# TODO: Verify if older versions than 2.6.4 can be built with Windows.`

Reply:
Ah, got it.